### ISSUE TITLE
Tiled Background with physics = crash

### ISSUE THREAD

**Issue by 4demon** [Scirra/Construct-3-bugs#4138]
I have a tiled background with physics. Had it for months. Suddenly Construct is very upset by the prospect :)

## Attach a .c3p

[Physics.zip](https://github.com/Scirra/Construct-3-bugs/files/5100295/Physics.zip)

## Steps to reproduce

1. Create a tiled background
2. Give it some physics
3. Play the layout

## Observed result

It crashes

## Expected result

It plays

## More details

Console show:

```
Uncaught (in promise) Assertion failed: bd->position.IsValid(), at: Box2D_v2.3.1/Box2D/Dynamics/b2Body.cpp,27,b2Body at Error
    at Ra (https://editor.construct.net/r212/behaviors/movements/physics/c3runtime/box2d.wasm.js:23:26)
    at ___assert_fail (https://editor.construct.net/r212/behaviors/movements/physics/c3runtime/box2d.wasm.js:59:222)
    at <anonymous>:wasm-function[613]:0x28143
    at _emscripten_bind_b2World_CreateBody_1 (<anonymous>:wasm-function[224]:0x1014f)
    at b._emscripten_bind_b2World_CreateBody_1 (https://editor.construct.net/r212/behaviors/movements/physics/c3runtime/box2d.wasm.js:245:262)
    at C.CreateBody (https://editor.construct.net/r212/behaviors/movements/physics/c3runtime/box2d.wasm.js:447:474)
    at PhysicsInstance._CreateBody (https://editor.construct.net/r212/behaviors/movements/physics/c3runtime/instance.js:7:402)
    at PhysicsInstance.PostCreate (https://editor.construct.net/r212/behaviors/movements/physics/c3runtime/instance.js:5:209)
    at BehaviorInstance.PostCreate (https://editor.construct.net/r212/preview/c3/objects/behaviorInstance.js:2:645)
    at Instance._CreateSdkInstance (https://editor.construct.net/r212/preview/c3/objects/instance.js:7:278)
```

**First affected release:** Worked in the last stable version... broke in r212.

**Comment by AshleyScirra** [comment]
It doesn't reproduce in my development copy, only after a build, when it's been processed by Closure Compiler. I suspect this is a Closure Compiler bug that's been affecting a few other things, so we'll need to wait for a new release of that to come out first before investigating further.

**Comment by 4demon** [comment]
@AshleyScirra the version I sent you isn't showing the issue?

In my tests, if I build the project from scratch, I'm able to use tiled backgrounds with physics... but when I open up a current project with that already in place, that's when it starts having issues.

**Comment by 4demon** [comment]
Seeing as there's a delay, I decided to do a work around instead, and stumbled upon something else that may be causing an issue with this.

When I went back to my project to remove the physics behaviour from the tiled backgrounds and replace it with something else, I noticed that at runtime only one of the tiled backgrounds appeared.

Upon looking closer into this, I found that the Construct highlight was showing them as all appearing on the left side, however the debug was still showing their X and Y positions as how I'd set them.

I have attached images to show what I'm seeing...

Is this a closure compiler bug, or something different?

[TiledErrors.zip](https://github.com/Scirra/Construct-3-bugs/files/5117876/TiledErrors.zip)
